packages/x-charts-pro/src/Heatmap/HeatmapTooltip/HeatmapTooltip.tsx

```diff
@@ -83,28 +82,18 @@ function DefaultHeatmapTooltipContent(props: Pick<HeatmapTooltipProps, 'classes'
 
   return (
     <ChartsTooltipPaper className={classes?.paper}>
-      <ChartsTooltipTable className={classes?.table}>
-        <thead>
-          <ChartsTooltipRow className={classes?.row}>
-            <ChartsTooltipCell className={classes?.cell}>{formattedX}</ChartsTooltipCell>
-            {formattedX && formattedY && <ChartsTooltipCell />}
-            <ChartsTooltipCell className={classes?.cell}>{formattedY}</ChartsTooltipCell>
-          </ChartsTooltipRow>
-        </thead>
-        <tbody>
-          <ChartsTooltipRow className={classes?.row}>
-            <ChartsTooltipCell className={clsx(classes?.markCell, classes?.cell)}>
-              <ChartsLabelMark type={markType} color={color} className={classes?.mark} />
-            </ChartsTooltipCell>
-            <ChartsTooltipCell className={clsx(classes?.labelCell, classes?.cell)}>
-              {seriesLabel}
-            </ChartsTooltipCell>
-            <ChartsTooltipCell className={clsx(classes?.valueCell, classes?.cell)}>
-              {formattedValue}
-            </ChartsTooltipCell>
-          </ChartsTooltipRow>
-        </tbody>
-      </ChartsTooltipTable>
+      <HeatmapTooltipAxesValue>
+        <span>{formattedX}</span>
+        <span>{formattedY}</span>
```

Comment:
If `HeatmapTooltipAxesValue` is a `Typography` component, would it make sense to use a `Typography` element here as well? 

What's the decision process to use MUI components vs native HTML ones?